apps/aecontract/test/aecontract_staking_contract_SUITE.erl

```diff
@@ -281,6 +283,39 @@ rewards(_Config) ->
 
         ok.
 
+    elct_penalties(_Config) ->
+        Trees0 = genesis_trees(),
+        TxEnv = aetx_env:tx_env(?DEFAULT_HEIGHT),
+        {ok, Trees1, #{res := {contract, AliceCt}}} =
+            new_validator_(pubkey(?ALICE), ?VALIDATOR_MIN, TxEnv, Trees0),
+        {ok, Trees2, #{res := {contract, BobCt}}} =
+            new_validator_(pubkey(?BOB), ?VALIDATOR_MIN, false, TxEnv, Trees1, undefined),
+
+        {ok, _, #{res := BaseEmptyPool}} = get_penalty_pool_(TxEnv, Trees2),
+        ?assertEqual(BaseEmptyPool, 0),
+
+        Rewards = [{{address, pubkey(?ALICE)}, -4 * ?AE}, {{address, pubkey(?BOB)}, 2 * ?AE}],
+        {ok, Trees25, #{res := ?UNIT}} =
```

Comment:
I expect a test in which Alice does something bad and therefore, the result of that is that she gets a penalty.
The test must make her do something bad and the system should respond with the penalty, that then the test should observe.